cinch/roles/jenkins_master/tasks/iptables.yml

```diff
@@ -10,6 +10,38 @@
     state: stopped
     enabled: false
 
+- name: set iptables to accept all icmp
+  iptables:
+    chain: INPUT
+    protocol: icmp
+    jump: ACCEPT
+
+- name: set iptables to accept all communication from lo interface
+  iptables:
+    chain: INPUT
+    in_interface: lo
+    jump: ACCEPT
+
+- name: set iptables to reject other communication to localhost
```

Comment:
Looking at some reference material, this rule may be closer to what we're porting over from our older code:

-A INPUT -d 127.0.0.0/8 ! -i lo -m comment --comment "reject local traffic not on loopback interface" -j REJECT --reject-with icmp-port-unreachable

Perhaps there is a reason to change it, but I'm not aware of a reason at this time.